dispatcher.py

```diff
@@ -7,7 +7,12 @@
 import yaml
 
 import multiprocessing
-from multiprocessing.queues import SimpleQueue
+try:
+    # Python 3.7+
+    from multiprocessing import SimpleQueue
+except ImportError:
+    # Python 2
+    from multiprocessing.queues import SimpleQueue
```

Comment:
I understand, it is from my old experimental [patch](https://github.com/tarantool/test-run/commit/2887259844431008bf868049ff564322010b841e), but, to be honest, I see no sense in this change.

First of all, the 'Python 3.7+' comment is misleading. The `SimpleQueue` class was exposed from the `multiprocessing` module since Python 3.3. See https://bugs.python.org/issue11836.

However it still [present](https://github.com/python/cpython/blob/a6192635f1e62af2bb8a435487ebb51800edd671/Lib/multiprocessing/queues.py#L10) in `multiprocessing.queues` and I don't heard anything regarding removal plans. So the change looks useless.

I didn't test the code, because now (on the first review iteration) I'm only looking around the patches. So, maybe, I don't know about some subtle difference. If there is some kind of this, it worth to mention it in the commit message.

Reply:
Without patch I got an error on start `test-run.py`:
```python
Traceback (most recent call last):
  File "../../test/test-run.py", line 233, in <module>
    status = main_parallel()
  File "../../test/test-run.py", line 148, in main_parallel
    res = main_loop_parallel()
  File "../../test/test-run.py", line 106, in main_loop_parallel
    dispatcher = Dispatcher(task_groups, jobs, randomize)
  File "/home/sergeyb/sources/MRG/tarantool/test-run/dispatcher.py", line 105, in __init__
    task_queue_disp = TaskQueueDispatcher(key, task_group, randomize)
  File "/home/sergeyb/sources/MRG/tarantool/test-run/dispatcher.py", line 378, in __init__
    self.result_queue = SimpleQueue()
TypeError: __init__() missing 1 required keyword-only argument: 'ctx'
```
There is a solution in a [Python bugtracker](https://bugs.python.org/msg230716):
```python
try:
    from multiprocessing import SimpleQueue
except ImportError:
    from multiprocessing.queues import SimpleQueue
```

Reply:
Okay, got it. However the comment 'Python 3.7+' still incorrect. Can I propose a wording?

```python
# SimpleQueue is available from multiprocessing.queues on
# all Python versions known at the moment of writting the code
# (up to 3.9).
#
# It was additionally exposed directly from the multiprocessing
# module since Python 3.3 ([1]).
#
# However the mandatory argument 'ctx'
# (see multiprocessing.get_context()) was added to the constructor
# of SimpleQueue from multiprocessing.queues since Python 3.4
# ([2]).
#
# So we should import SimpleQueue from multiprocessing on
# Python 3.3+ (and must to do so on Python 3.4+) to uniformly
# instantiate it (without constructor arguments).
#
# [1]: https://bugs.python.org/issue11836
# [2]: https://bugs.python.org/issue18999
try:
    # Python 3.3+
    from multiprocessing import SimpleQueue
except ImportError:
    # Python 3.2 and below (including Python 2)
    from multiprocessing.queues import SimpleQueue
```

Reply:
Replaced `Python 3.7+` with `Python 3.3+` and added wording proposed by you.